AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Organizations/OrganizationIdQuery.cs

```diff
@@ -1,10 +0,0 @@
-﻿using AllReady.Models;
-using MediatR;
-
-namespace AllReady.Areas.Admin.Features.Organizations
-{
-    public class OrganizationIdQuery : IAsyncRequest<int>
-    {
-        public int? ItineraryId { get; set; }
```

Comment:
@stevejgordon I can revert for now and and name as you originally did. The renaming of the message/handler was b/c the the original name of OrganizationIdQuery was confusing to me at first.Now, when I look at my changes, the initial message/handler name is fine, and my renaming doesn't really make sense. I don't know what i was thinking :confused: 

As far as extending the query message to use nullable ints for all the main look-ups, it would help in re-usability, but could be very confusing to the developer trying to utilize the message. A command should have one logical handler in the system. Although this design still adheres to that, the developer still needs to look at the conditional code in the handler to understand the ramifications of including or dis-cluding values from the properties on a given message. This obfuscates the intent of the message and violates SRP in the handler.

IMO, I'd rather work with/write unit tests for three smaller message/handler classes that all do almost the same thing, then one larger handler that I need to figure out what property on that command should be nullable based on which execution path I want to run in the handler.

Either way, I'll let you make this decision, b/c you understand these UC's at a higher level than I do, and understand how they work with the rest of the system better than I do.